### ISSUE TITLE
No Jupyter notebook available

### ISSUE THREAD

**Issue by dpasch01** [saref/clusterability-index#1]
Greetings,

Could you please make the notebook available?

Thank you.

**Comment by saminaref** [comment]
Yes, it will be made publicly available upon acceptance of the paper.

**Comment by saref** [comment]
Yes, it will be made publicly available upon acceptance of the paper.

**Comment by saref** [comment]
Dear Demetris Paschalides @dpasch01 , The jupyter notebook is publicly available now. Please feel free to write here or contact me by email if you have any questions.
